### PR TITLE
REST in Science (RIS)

### PR DESCRIPTION
**Pull request by asherpasha**
Hi, I am Asher. I talked about RESTful web services during U of T Scientific Coder meeting. Could you please add my talk too :) Thanks.

### CODE REVIEW

File: lessons/REST/lesson.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+layout: page
+title:
+visible: false
+---
+<!-- change visible to true if you want it on the site -->
```
Comment (QuLogic) [inline lessons/REST/lesson.md:6]: Please read this comment ;)

File: lessons/REST/lesson.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+layout: page
+title:
```
Comment (QuLogic) [inline lessons/REST/lesson.md]: Title missing.

File: lessons/REST/lesson.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+visible: false
+---
+<!-- change visible to true if you want it on the site -->
+
+# RESTful Web Services in Science
+
+ - **Authors**: Asher Pasha
+ - **Research field**: Bioinformatics
+ - **Lesson topic**: Get data from RESTful web services
+ - **Lesson content URL**: <link>
```
Comment (QuLogic) [inline lessons/REST/lesson.md]: Please add the link here (just see what other lessons do.)

File: lessons/REST/Presentation.tex [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+
+\begin{document}
+
+	\frame{\titlepage}
+
+	\begin{frame}
+		\frametitle{CGI Programs}
+		\begin{figure}[!htb]
+			\centering
+			\includegraphics[width=2in]{cgiarch.eps}
```
Comment (QuLogic) [inline lessons/REST/Presentation.tex:22]: This (and all other) image is not included.

**Comment by QuLogic** [comment]
Please add the compiled PDF as well so that everyone doesn't need to re-create it themselves.

**Comment by asherpasha** [comment]
I have made the recommended updates. How about now? Comments? Suggestions?

**Comment by lwjohnst86** [comment]
Only other thing is to include the `cgiarch.eps` image file into the code and you are all good!

**Comment by QuLogic** [comment]
> `cgiarch.eps` image file

plus the other images.

**Comment by asherpasha** [comment]
I have added the images. Also, I have tested the tex file, and it compiles.

**Comment by lwjohnst86** [comment]
Excellent, thanks

**Pull request merged**